style.css

```diff
@@ -174,3 +174,525 @@ header nav div .menu-bar {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 114px;
+}
+
+.experience1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  padding-bottom: 12px;
+  height: 124px;
+}
+
+.section-title1 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 0;
+  padding-top: 12px;
+  width: 295px;
+  height: 80px;
+  white-space: pre-line;
+}
+
+.section3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 552px;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 114px;
+}
+
+.img-sec3 {
+  display: none;
+  height: 0;
+  width: 0;
+}
+
+.section4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 612px;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 114px;
+}
+
+.section5 {
+  margin-top: 114px;
+  padding: 114px 24px;
+  margin-left: 0;
+  margin-right: 0;
+  width: 327px;
+  height: 986.85px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border-top-right-radius: 70px;
+}
+
+.top {
+  margin-top: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-heading {
+  width: 327px;
+  height: 30px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  white-space: pre-line;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-description {
+  width: 327px;
+  height: 144px;
+  top: 113px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.order2 {
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.resume {
+  box-sizing: border-box;
+  padding: 12px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: #fff;
+}
+
+.skills-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-items: flex-start;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  height: 72px;
+  margin-left: 0;
+}
```

Comment:
Menubar Title & Dropdown arrow icon are not aligned as per FIGMA design.
Add `align-items: baseline;` to align the title of the menubar and dropdown arrow icon.
![image](https://user-images.githubusercontent.com/61361037/212448176-eab6a778-3b1e-496d-bcff-98c3269f2513.png)